dj_swud/app/controllers/dj_swud_controller.rb

```diff
@@ -0,0 +1,67 @@
+
+class DjSwudController < ApplicationController
+  def index
+  end
+
+  def random_word
+  	response = 'http://randomword.setgetgo.com/get.php'
```

Comment:
response = HTTParty.get 'http://randomword.setgetgo.com/get.php' # this makes the request to the API
word = response.parsed_response #this grabs just the word from the response from the API